### ISSUE TITLE
reconnect command fails and autoreconnect doesn't seem to work

### ISSUE THREAD

**Issue by Mass1milian0** [jubiman/DeviousDiscord#15]
it gives me this error when i run devious admin reconnect also the autoreconnect timer doesn't seem to work

```
java.lang.NullPointerException: Cannot invoke "java.util.TimerTask.cancel()" because "this.TIMER_TASK" is null
at com.jubiman.devious.discord.forge.v1_18_2.deviousdiscord.network.WebSocketConnection.reconnect(WebSocketConnection.java:163) ~[deviousdiscord-FORGE1.18.2-v1.1.6.jar%2392!/:com.jubiman.devious.discord.forge.v1_18_2] {re:classloading}
at com.jubiman.devious.discord.forge.v1_18_2.deviousdiscord.DeviousDiscord.lambda$onCommandsRegister$1(DeviousDiscord.java:47) ~[deviousdiscord-FORGE1.18.2-v1.1.6.jar%2392!/:com.jubiman.devious.discord.forge.v1_18_2] {re:classloading}
at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:263) ~[brigadier-1.0.18.jar%2325!/:?] {}
at net.minecraft.commands.Commands.m_82117_(Commands.java:234) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:mixin,re:classloading,pl:mixin:APP:ftbranks-common.mixins.json:CommandsMixin,pl:mixin:A}
at net.minecraft.server.dedicated.DedicatedServer.m_139665_(DedicatedServer.java:341) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:327) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:infernal-expansion.mixins.json:common.MixinMinecraftServer,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:infernal-expansion.mixins.json:common.MixinMinecraftServer,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23210!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:infernal-expansion.mixins.json:common.MixinMinecraftServer,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:byg.mixins.json:access.MinecraftServerAccess,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}

```

**Comment by jubiman** [comment]
This is a huge oversight of mine. Will be fixed in 1.2.0 asap.

Basically I don't always create a TIMER_TASK, as if the initial connection fails, it uses slightly different logic to reconnect (as it does not have to check whether it is connected or not, but it cancels the task instantly after calling reconnect(), which generates a null pointer exception, since it tries to cancel TIMER_TASK, which has not been set

**Comment by jubiman** [comment]
Fixed in v1.2.0
